sass/_header.scss

```diff
@@ -3,125 +3,114 @@ Header
 --------------------------------------------------------------*/
 
 .header {
-    @include rem(font-size, 24px);
-    color: #fff;
-	padding: 0 20px;
+  @include rem(font-size, 24px);
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.